### PR TITLE
Mobile contact me

### PR DESCRIPTION
**Pull request by Kaghenimbale**
- [ ]  Add a Name field that accepts up to 30 chars.
- [ ]  Add an Email field that accepts only emails.
- [ ]  Add a message field that accepts up to 500 chars.
- [ ]  Add in all fields placeholder text.
- [ ]  Add in all fields the correct type.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,14 +174,25 @@ <h3>LET'S CONNECT</h3>
...
+                    </label>
+                    <label for="message">
+                        <textarea name="message" class="msg" maxlength="500" placeholder="Write your message here"></textarea>
+                    </label>
+                    <div class="button">
+                      <button type="submit" class="btn last-btn">Get in touch</button>
+                    </div>
+                </fieldset>
+
+            </form>
```
Comment (anagudelogu) [inline index.html]: You did a good job working on this milestone! 💯 However, there's still some room to improve.

- [ ] 🚧 According to the requirements, you should use the `Formspree` service to collect data, please refer to their [documentation](https://formspree.io/html/) and implement this feature in your form! 🤓 🔧 

Overall, the design is good, but there are small issues that you could easily solve in order to make it even better. 🔧 

|Template|Your design|
|---|---|
| image | image |

- [ ] 🚧 Please make sure to use the correct background image, as you can see, it is different from the one provided in the Figma template. 🔧 

- [ ] 🚧 Consider fixing the typo in the title `"Contacts"` ==> `"Contact"`. 🔧 

- [ ] 🚧 Please add some padding to your input fields, the text is very close to the edges! 🔧 

Applying those changes will improve the look and feel of this section of your page, so try to spend some time working on them! 🤓 🛠️

File: index.html [hunk 2]
```diff
@@ -174,14 +174,25 @@ <h3>LET'S CONNECT</h3>
...
+            <form action="#" method="post">
+                <fieldset class="contact-form">
+                    <label for="username">
+                        <input type="text" class="input-contact" required name="username" maxlength="30" placeholder="Username">
+                    </label>
+                    <label for="email">
+                        <input type="email" class="input-contact" required name="email" placeholder="name@domain.com">
+                    </label>
+                    <label for="message">
+                        <textarea name="message" class="msg" maxlength="500" placeholder="Write your message here"></textarea>
```
Comment (anagudelogu) [inline index.html]: - [ ] Good job applying the `maxlength` attribute to the `textarea` element! 🚀 Please make sure to make this field required too, according to the requirements, all the inputs should be required. 🤓 🔧 

From requirements:

image

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @Kaghenimbale, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

- [ ] 🚧 Please consider adding a LICENSE to your project, at the moment, the link you provided in the README redirects to an external website. I suggest you follow [these steps](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) from the official GitHub docs in order to add a LICENSE to your project. Don't forget to link it in your README! 🤓 🔧  

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -174,14 +174,29 @@ <h3>LET'S CONNECT</h3>
...
+                    <label for="message">
+                        <textarea name="message" class="msg" required maxlength="500" placeholder="Write your message here"></textarea>
+                    </label>
+                    <div class="button">
+                      <button type="submit" class="btn last-btn">Get It Touch</button>
+                    </div>
+                </fieldset>
+                <img src="image/Ellipse 5.png" class="ellipse-5" alt="ellipse-5">
+                <img src="image/Rectangle 7.png" class="rectangle-7" alt="rectangle-7">
+            </form>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Good job implementing the changes suggested by the previous reviewer. However, your form is not been successfully submitted. Kindly follow the [Guide here](https://formspree.io/html/), and make sure you don't miss anything. 

# Output:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Kaghenimbale   👋 


# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing some of the suggested changes from the previous reviewer, but there are still some which aren't been addressed yet.



Highlights
✅All linters are checked
✅ Descriptive commit messages 

Required Changes ♻️

- [ ] I must commend you for adding the form as suggested 👏  , but presently, there's a bug that doesn't get the form to submit the input text. Check the comment section for a more detailed review.

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by pinkmoon25 — APPROVED** [review]
Hi @Kaghenimbale , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
Integrate formspree :heavy_check_mark: 
form implementation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**